packages/web-react/src/types/visuallyHidden.ts

```diff
@@ -0,0 +1,6 @@
+import { StyleProps } from './shared';
+
+export interface VisuallyHiddenProps extends StyleProps {
+  children: string;
```

Comment:
Are we sure about this?

Reply:
yes, that was mistake, should be fixed now